src/components/radio-group/radio-group.ts

```diff
@@ -39,6 +59,48 @@ export default class SlRadioGroup extends LitElement {
     });
   }
 
+  getAllRadios(): SlRadio[] {
+    return [...this.querySelectorAll('sl-radio')];
+  }
+
+  handleKeyDown(event: KeyboardEvent) {
+    if (['ArrowUp', 'ArrowDown', 'ArrowLeft', 'ArrowRight'].includes(event.key)) {
+      const radios = this.getAllRadios().filter(radio => !radio.disabled);
+      const currentIndex = radios.findIndex(el => el.checked);
+
+      const incr = ['ArrowUp', 'ArrowLeft'].includes(event.key) ? -1 : 1;
+      let index = currentIndex + incr;
+      if (index < 0) index = radios.length - 1;
+      if (index > radios.length - 1) index = 0;
+
+      this.getAllRadios().map(radio => (radio.checked = false));
```

Comment:
I think `forEach` would be better than `map` here.

Reply:
Is this a performance assumption? I tend to use `map` over `forEach` since the result of `getAllRadios()` is an array.

Reply:
Nope. The result is not used, so `mapping` does not add any usefulness. `forEach` and `map` both iterate over the array elements. The latter answers a new array with the result of each iteration as new value for the array. Ie `[ 1, 2, 3 ].map(function(x) { return x * x; })` will result in `[ 1, 4, 9 ]`. `forEach` always returns `undefined`.

Reply:
I do think performance is slightly worse with `map` since a new array structure is created. For `forEach` this is not necessary. Not sure if this is noticeable in real live.